dev/k3d-clean

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+k3d cluster list -o json | \
+    jq -r '.[].name' | \
+    xargs -I{} k3d cluster delete {}
```

Comment:
Is this different from ` k3d cluster delete --all`?

Reply:
No, and that's a good point. But now I'm considering the suggestion of @weyfonk to make it only delete clusters that are set in environment variables, which will make it look similar to as it currently does.